bash.md

```diff
@@ -5,11 +5,146 @@ The shell scripting language of choice is Bash.  Bash should be used for start s
 
 For the most part, follow [Googles shell coding standards](https://google.github.io/styleguide/shell.xml) with the **exception of using four spaces** (not two as [specified in the standard](https://google.github.io/styleguide/shell.xml#Indentation)).
 
-Determining the containing directory
-------------------------------------
+# Files
 
-To get the containing directory of the script as done in all service start scripts use:
+## Naming
+The preferred option is to omit file extensions such as `.sh`
+
+**Good**
```

Comment:
Please add blank line after headings when followed by code blocks - not all editors will parse correctly without a blank line.